tests/unit/via/app_test.py

```diff
@@ -43,29 +42,6 @@ def test_settings_are_configured_from_environment_variables(pyramid_settings):
         assert settings[key] == value
 
 
-@pytest.mark.usefixtures("os")
-def test_app(configurator, pyramid, pyramid_settings):
-    create_app()
-
-    expected_settings = dict(
-        {"h_pyramid_sentry.filters": SENTRY_FILTERS}, **pyramid_settings
-    )
-    expected_settings["data_directory"] = Path(pyramid_settings["data_directory"])
-
-    pyramid.config.Configurator.assert_called_once_with(settings=expected_settings)
-    configurator.make_wsgi_app.assert_called_once_with()
-
-
-@pytest.fixture
-def configurator(pyramid):
-    return pyramid.config.Configurator.return_value
-
-
-@pytest.fixture(autouse=True)
-def pyramid(patch):
-    return patch("via.app.pyramid")
```

Comment:
This test was just annoying, using mocks instead of using the `pyramid.testing` stuff properly. With this PR's changes to `app.py` the test now crashes with:

    AttributeError: Mock object has no attribute 'registry'

I really don't think there's much value in testing the `create_app()` function: there's no logic in it, and if it's broken you'll know pretty fast. So I've just added a coverage ignore to it